pom.xml

```diff
@@ -44,6 +44,11 @@
 
     <dependencyManagement>
         <dependencies>
+            <dependency>
+                <groupId>com.sun.xml.parsers</groupId>
```

Comment:
Please check the license version and vulnerability if we are adding this to our IS pack?